src/components/schedule-page/table/schedule/schedule-parts/base-cell/base-cell.component.tsx

```diff
@@ -172,7 +175,7 @@ export function BaseCellComponent({
         <div className={"content " + hasNextClass + " " + keepOnClass} data-cy="highlighted-cell">
           {isPointerOn && !isBlocked && (
             <InputComponent
-              className="cell-input"
+              className={"cell-input " + moreMarginClasss}
```

Comment:
tutaj to samo co Tomek pisał, użycie classNames

Reply:
starałam się do tego zastosować, ale moim zdaniem teraz to wygląda mniej czytelnie z pktu patrzenia na kod